### ISSUE TITLE
IT Support: [Web phone number issue]

### ISSUE THREAD

**Issue by Vaishnav1397** [cityofaustin/atd-data-tech#18433]
St Elmo
## Device & User Information (first name and last initial, no last names)
Phone & Monika, S
## What's the problem?
User requested and explained the scenario for one of their active recently moved user phone number has not been updated, but the supervisor has been changed.
## What did you do to resolve the issue
Have checked with the user, got to know on the issue, that is a network support and there is another ticket in SN which was been raised, in order to get the user information. So user waiting on update through SN team.
